### ISSUE TITLE
P02s phosphorus BODCNVS-1722

### ISSUE THREAD

**Issue by MeganAnneFrench** [nvs-vocabs/EMODnetChemVocabs#36]
Hi,
I have attached an Excel file with some suggested changes to P02 and P36 codes relating to P01s for phosphorus (based on the file created by Dick in November 2022 - the non-shaded columns). The yellow shaded columns show the suggested changes and the grey shaded columns are notes. The suggested P02 change relates to a deprecated code: "Phosphorus concentrations in sediment DEPRECATED", but it is unclear why this was deprecated. As the other main nutrient-type element, nitrogen has P02s of "Nitrogen concentrations in sediment" (inorganic nitrogen is mapped here), "Dissolved total and organic nitrogen concentrations in the water column", and "Particulate total and organic nitrogen concentrations in the water column", whereas the active P02s for phosphorus are just "Dissolved total or organic phosphorus concentration in the water column" and "Particulate total and organic phosphorus concentrations in the water column". Phosphorus and nitrogen should be treated in the same way.
Megan
[P02s phosphorus.xlsx](https://github.com/nvs-vocabs/EMODnetChemVocabs/files/10907837/P02s.phosphorus.xlsx)

**Comment by mmolinajack** [comment]
The phosphorous should be considered similar to the nitrogen and now there are inconsistencies in this sense.

**Comment by MeganAnneFrench** [comment]
yes, it is ok, I forget to write that and I am amending it!

**Comment by gwemon** [comment]
Investigating why http://vocab.nerc.ac.uk/collection/P02/current/PXSD/ was deprecated. I agree that it does not look right.

**Comment by gwemon** [comment]
@MeganAnneFrench  As you said, we do have [Nitrogen concentrations in sediment](http://vocab.nerc.ac.uk/collection/P02/current/NTSD/) so one would expect an equivalent category for phosphorus. I can't think of or track back to a reason why PXSD was deprecated (it was deprecated in 2012) so I have re-instated it as it does not really make sense to have it deprecated and replacedBy [MIN](http://vocab.nerc.ac.uk/collection/P02/current/NTSD/) .

**Comment by MeganAnneFrench** [comment]
Ok, thanks Gwen!

**Comment by gwemon** [comment]
I have also reviewed the mappings and all "concentration of phosphorus in sediment" codes now point to tp PXSD and not RMIN or MTSD. 
The only "phosphorus" related codes I kept under RMIN are the "geological" phosphorus because these would not belong to "eutrophication/fertilisation" theme. These codes are:
GEOLST22	Concentration of total phosphorus {total_P CAS 7723-14-0} per unit dry weight of geological sample
MAJO0010	Proportion of phosphorus (expressed as phosphorus pentoxide) {P_as_P2O5 CAS 7723-14-0} per unit weight of sediment
MAJO0021	Proportion of phosphorus (expressed as phosphorus pentoxide) {P_as_P2O5 CAS 7723-14-0} per unit weight of rock
MAJO0034	Proportion of phosphorus (expressed as phosphorus pentoxide) {P_as_P2O5 CAS 7723-14-0} per unit weight of geological sample
 
The changes will be live after our overnight refresh.
